sed/tests/test.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
```

Comment:
This `test.sh` is out of date. We should update this to be more in line with the new testing approach/standard that we have adopted. See examples in plans such as: hugo, terraform, nginx, hapropxy etc. The same applies for the sourceing in `test.bats`, which is no longer needed when using the new approach.

Reply:
This `test.sh` seems to meet the current testing standard and I don't see any sourcing in `test.bats`

Does this comment refer an older revision of this PR (it did previously have an initial attempt that predated the standard being decided), or am I missing something?

Reply:
Ah yes I did change the tests a day after this comment was left. I think the line that @predominant commented on remained unchanged, so Github doesn't consider this comment outdated.